index.html

```diff
@@ -0,0 +1,204 @@
+<!doctype html>
+	<html>
+		<head>
+			<meta charset="utf-8">
+			<meta http-equiv="X-UA-Compatible" content="IE=edge">
+			<title></title>
+			<meta name="description" content="">
+			<meta name="viewport" content="width=device-width, initial-scale=1">
+			<link href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css" rel="stylesheet">
+			<link rel="stylesheet" type="text/css" href="styles.css">
+		</head>
+		<body>
+			<nav class='navbar navbar-fixed-top'>
+				<div class="container">
+					<div class="navbar-header">
+						<a href="#">
+							<h1>Danebook</h1>
+						</a>
+					</div>
+					<div class="navbar-collapse collapse">
+						<form class="navbar-form navbar-right">
+							<div class="form-group">
+								<input class="form-control" type='text' placeholder='Email'></input>
+							</div>
+							<div class="form-group">
+								<input class="form-control" placeholder='password' type='password'></input>
+							</div>
+							<button class="btn btn-primary bordered">Log In</button>
+						</form>
+					</div>
+				</div>
+			</nav>
+
+			<div id='homepage-left' class="container">
+				<div class="nav-spacer"></div>
+				<div class="row">
+					<h1 style='color: black'><b>Connect with all your friends</b></h1>
+				</div>
+				<div class="homepage-list">
+					<div class="inner-spacer"></div>
+					<div class="row">
+						<p><big>- See photos and updates in your news feed.</big></p>
+					</div>
+					<br>
+					<div class="row">
+						<p><big>- Post your status for the world to see from your profile.</big></p>
+					</div>
+					<br>
+					<div class="row">
+						<p><big>- Get in touch with your friends by "friending" them.</big></p>
+					</div>
+					<br>
+					<div class="row">
+						<p><big>- Like things because you're a positive person!</big></p>
+					</div>
+				</div>
+			</div>
+			<div id="homepage-right" class="container">
+				<div class="row">
+					<h1><big>Sign Up</big></h1>
+				</div>
+				<br>
+				<div class="row">
+					<form>
+						<div class="col-md-4">
+							<input class="full-input" type='text' placeholder='First Name'></input>
+						</div>
+						<div class="col-md-4">
+							<input class="full-input" type='text' placeholder='Last Name'></input>
+						</div>
+						<br>
+						<br>
+						<br>
+						<div class="col-md-8">
+							<input class="full-input" type='text' placeholder='Your Email'></input>
```

Comment:
Might be worth it to move this into a new "row" division. Having more than 12 columns inside a row might cause problems with bootstrap